packages/framework/theming-react-native/src/NativeModule/ThemingModuleHelpers.ts

```diff
@@ -46,15 +45,12 @@ export function createThemingModuleHelper(themingModule?: OfficeThemingModule, e
   let _hostTheme = themingModule.initialHostThemeSetting || '';
   emitter &&
     emitter.addListener('onPlatformDefaultsChanged', (args: PlatformDefaultsChangedArgs) => {
-      _hostTheme = (args && args.hostThemeSetting) || _hostTheme;
```

Comment:
You can't rename this without native theming module changes:

```
        dynamic params = dynamic::array("onPlatformDefaultsChanged", dynamic::object("hostThemeSetting", getThemeSetting()));
        instance->callJSFunction("RCTDeviceEventEmitter", "emit", std::move(params));
```
That creates a JSON object `{ hostThemeSetting: 'Foo' }`

Sorry, name is probably clear, I think the native code calls it the Theme Setting so I was probably borrowing that nomenclature

Reply:
Thankfully we can name it whatever we want in our pure JS interfaces, so this is really just an implementation detail